spec/JSON-RPC.md

```diff
@@ -3063,6 +3064,70 @@ No parameters
 
 [Back to **List of methods**](#list-of-methods)
 
+## devel_getPeerBestBlockHashes
+
+Get IP address and best block hash of each peer.
+
+### Params
+
+No parameters
+
+### Returns
+
+{ SocketAddress: `string`, BlockHash: `H256`}[]
```

Comment:
This type does not match the example.
What you said:
```js
[ 
{ "SocketAddress": "xxx", "BlockHash": "yyy"},
{ "SocketAddress": "xxx2", "BlockHash": "yyy2"}, ...
]
```

However, in the example:
```js
[["1.2.3.4:3485", "0x56642f04d519ae3262c7ba6facf1c5b11450ebaeb7955337cfbc45420d573077"]]
```

Reply:
Okay.  Should I introduce new type such as SocketAddressAndHash, or tuple type is sufficient?

Reply:
IMO, tuple type is sufficient.

Reply:
Then the return type should be [ 'string', 'H256'][]?

Reply:
It looks good to me.